pkg/collector/corechecks/checkbase.go

```diff
@@ -214,6 +214,10 @@ func (c *CheckBase) ConfigSource() string {
 	return c.source
 }
 
+func (c *CheckBase) Loader() string {
+	return "Go"
```

Comment:
@pgimalac Drive-by note: Should we make sure that these are all lowercase to avoid issues with capitalization/filtering on the backend?

Reply:
I lowercased the strings, and added a comment on the `Loader` method to make it clear the strings were used in tags so had to follow the constraints.

Does that work for you, or were you thinking of converting whatever is returned by `Loader` instead ? I don't believe we have a conversion function... (so using `strings.Lowercase` would work for those specific strings but not in general)